app/assets/stylesheets/application.scss

```diff
@@ -53,24 +57,20 @@
 .entries {
   background-color: #ffffff;
   list-style: none;
-  margin: 15px;
+  margin: 0 20px;
+  padding: 10px 0;
   text-align: center;
   border-bottom: 1px solid #ccc;
 }
 
 .stats {
-  background-color: #ffffff;
-  text-align: center;
+  list-style: none;
   margin: 0;
-  padding: 0;
 }
 
 .map-container {
   height: 400px;
-  border-color: #fff;
-  border-style: solid;
-  box-shadow: 2px 2px 10px #B1B1B1;
-  border-width: 7px;
+  box-shadow: 5px 5px 5px #b1b1b1;
```

Comment:
Color literals like `#b1b1b1` should only be used in variable declarations; they should be referred to via variable everywhere else.